native_client/deepspeech.cc

```diff
@@ -265,6 +265,21 @@ StreamingState::processBatch(const vector<float>& buf, unsigned int n_steps)
 int
 DS_CreateModel(const char* aModelPath,
                ModelState** retval)
+{
+  return DS_CreateModel_(aModelPath, false, retval);
+}
+
+int
+DS_CreateModelFromBuffer(const std::string &aModelBuffer,
+                         ModelState** retval)
+{
+  return DS_CreateModel_(aModelBuffer, true, retval);
+}
+
+int
+DS_CreateModel_(const std::string &aModelString,
```

Comment:
Not sure how this is implemented, we go straight from `const char*` to `std::string &`